### ISSUE TITLE
Profile Menu and Profile Builder Translation

### ISSUE THREAD

**Issue by JuliaKiselyova** [CrocoBlock/suggestions#3890]
Profile builder can't be translated at the moment

Unfortunately, there is no way to make the account in a few languages with the Profile builder.

**Perhaps it would be worthwhile to add this feature in future updates:)**

**Comment by CodeByFlo** [comment]
Yes this is essential! Please make sure the profile builder can be translated with WPML, because without it the profile builder is almost useless to us. Thank you so much for prioritizing this 👍
